backend/src/api/mining/mining.ts

```diff
@@ -31,7 +33,7 @@ class Mining {
    */
   public async $getHistoricalBlockFees(interval: string | null = null): Promise<any> {
     return await BlocksRepository.$getHistoricalBlockFees(
-      this.getTimeRange(interval),
+      this.getTimeRange(interval, 5),
```

Comment:
@nymkappa what are you trying to do with this line?

I'm a bit confused given the current implementation of `getTimeRange`:

https://github.com/mempool/mempool/blob/a97675c5388e14edddfe87930112e75e7bd7f5c8/backend/src/api/mining/mining.ts#L531-L544

Reply:
It reduces the resolution x5 for the block fees charts to be more visually pleasing (less noisy)

This is what the chart look like without that scaling.

<img width="1701" alt="Screen Shot 2022-07-12 at 8 46 47 AM" src="https://user-images.githubusercontent.com/9780671/178426295-a32d9930-d12c-4e02-85d4-88002bd639d4.png">

VS now

<img width="1711" alt="Screen Shot 2022-07-12 at 8 47 35 AM" src="https://user-images.githubusercontent.com/9780671/178426379-fe689af1-1a11-4043-ba0f-211ea915747b.png">